core-services/image-mirroring/openshift/mapping_origin_4_7

```diff
@@ -92,6 +92,7 @@ registry.svc.ci.openshift.org/origin/4.7:docker-registry quay.io/openshift/origi
 registry.svc.ci.openshift.org/origin/4.7:egress-dns-proxy quay.io/openshift/origin-egress-dns-proxy:4.7 quay.io/openshift/origin-egress-dns-proxy:4.7.0 quay.io/openshift/origin-egress-dns-proxy:latest
 registry.svc.ci.openshift.org/origin/4.7:egress-http-proxy quay.io/openshift/origin-egress-http-proxy:4.7 quay.io/openshift/origin-egress-http-proxy:4.7.0 quay.io/openshift/origin-egress-http-proxy:latest
 registry.svc.ci.openshift.org/origin/4.7:egress-router quay.io/openshift/origin-egress-router:4.7 quay.io/openshift/origin-egress-router:4.7.0 quay.io/openshift/origin-egress-router:latest
+registry.svc.ci.openshift.org/origin/4.7:egress-router-cni quay.io/openshift/origin-egress-router-cni:4.7 quay.io/openshift/origin-egress-router-cni:4.7.0 quay.io/openshift/origin-egress-router-cni:latest
```

Comment:
Is the egress-router-cni image specific to ovn-k? If so, should we name that as such? To not confuse with the sdn one (if any).

Reply:
it's actually not, even if it would be used only for that in the beginning. Theoretically it should also work with openshift-sdn, even though I won't deprecate the existing egress-router yet. TL; DR. I'd avoid mapping it directly to ovn-k for now.